lib/metrics/events.js

```diff
@@ -98,7 +100,12 @@ module.exports = log => {
         }
 
         return emitFlowEvent(event, request, data)
-          .then(() => {})
+      })
+      .then(metricsContext => {
+        amplitude(event, request, data, metricsContext)
+        if (metricsContext && event === metricsContext.flowCompleteSignal) {
+          amplitude('flow.complete', request, data, metricsContext)
```

Comment:
In fact, I wonder if it's possible to move the "if (event === data.flowCompleteSignal)" logic out of `emitFlowEvent` and just do it in one place here, after the other events have been emitted.